src/server/Session.ts

```diff
@@ -1,6 +1,8 @@
 import { v4 as createUuid } from 'uuid'
 import * as ws from 'ws'
 
+const EMPTY_SESSION_TIMEOUT = 90
```

Comment:
I think there should be either a comment or a different variable name here to clarify the timeout is in seconds, not milliseconds. Just to be extra clear